### ISSUE TITLE
`Mut` export misses from ECS prelude

### ISSUE THREAD

**Issue by berkus** [bevyengine/bevy#123]
Code from the main page 

```
use bevy::prelude::*;

....

// this system runs on each entity with a Position and Velocity component
fn movement(position: Mut<Position>, velocity: &Velocity) {
    position.0 += velocity.0;
}
```

does not compile, citing `Mut` unknown.

Adding explicit `use bevy::ecs::Mut;` fixes it. It is apparently missing from prelude.

**Comment by berkus** [comment]
It also misses `mut` before position argument.

**Comment by cart** [comment]
Nice catch. Thanks!

**Comment by Halfwhit** [comment]
This looks like something a newbie like me would be interested in doing

**Comment by cart** [comment]
Closed by #172 and https://github.com/bevyengine/bevy-website/pull/9

**Comment by berkus** [comment]
👍
